FirebaseTestProject/Models/Post.swift

```diff
@@ -23,6 +23,39 @@ struct Post: FirebaseConvertable {
         self.timestamp = Date()
     }
     static let testPost = Post(title: "Title", text: "Content", author: "First Last")
+    
+    func contains(_ str:String) -> Bool {
+        // Probably Works -- VERY ARBITRARY (works for any keys)
+        let stringValues = self.jsonDict.values.map { val -> String in
+            if let value = val as? String {
+                return value
+            }
+            // We don't want to search on ANY UUIDs
+            if let value = val as? UUID {
+                return ""
+            }
```

Comment:
LMK: Are we able to remove this? I think the `return ""` at the end of this closure should take care of any UUID values—am I wrong?